app/views/discovered_hosts/_discovered_host_modal.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="modal fade" id="<%= "fixedPropertiesSelector-#{host.id}" %>" tabindex="-1" role="dialog" aria-labelledby="fixedPropertiesSelectorLabel">
   <div class="modal-dialog" role="document">
     <div class="modal-content">
+      <%= form_for host, :url => edit_discovered_host_path(host), :method => :get do |f| %>
```

Comment:
how about moving this form_for to be above `modal-body` div? I'm not sure if the header should be inside the form, plus please indent the correlated `end`

Reply:
I think this should do the trick!